packages/cli/uni-builder/src/types.ts

```diff
@@ -380,7 +381,9 @@ export type UniBuilderPlugin = {
 export type UniBuilderConfig = {
   dev?: RsbuildConfig['dev'];
   html?: RsbuildConfig['html'];
-  output?: RsbuildConfig['output'];
+  output?: Omit<NonNullable<RsbuildConfig['output']>, 'polyfill'> & {
```

Comment:
I think we also need to convert the `ua` value to `off`, to make sure that no polyfill will be injected by Rsbuild

Reply:
Nice catch, done.